CONTRIBUTING.md

```diff
@@ -31,7 +31,7 @@ make test
 
 ## Dependency management
 
-The K8s athenz SIA project uses [Go modules](https://golang.org/cmd/go/#hdr-Modules__module_versions__and_more) to manage dependencies on external packages. This requires a working Go environment with version 1.18 or greater installed.
+The K8s athenz SIA project uses [Go modules](https://golang.org/cmd/go/#hdr-Modules__module_versions__and_more) to manage dependencies on external packages. This requires a working Go environment with version 1.20 or greater installed.
```

Comment:
```suggestion
The K8s athenz SIA project uses [Go modules](https://golang.org/cmd/go/#hdr-Modules__module_versions__and_more) to manage dependencies on external packages. This requires a working Go environment with version 1.20 or later installed.
```

Reply:
@mlajkim resolve in new PR: https://github.com/AthenZ/k8s-athenz-sia/pull/9#issue-1641688535